make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -207,3 +207,8 @@ UPDATE quota_usage SET used = used - 'count';
 /* make Clair and Trivy as reserved name for scanners in-tree */
 UPDATE scanner_registration SET name = concat_ws('-', name, uuid) WHERE name IN ('Clair', 'Trivy') AND immutable = FALSE;
 UPDATE scanner_registration SET name = split_part(name, '-', 1) WHERE immutable = TRUE;
+
+/*update event types in table 'notification_policy'*/
```

Comment:
please format the code so it's easier to read.

Reply:
> please format the code so it's easier to read.

will be done